js/htmldiff.js

```diff
@@ -41,7 +41,8 @@
     }
 
     function isTag(token){
```

Comment:
Can you add jsdocs here so it's clear that this is now returning the tag name if the token is a tag.